### PR TITLE
Unit test for #1093

### PR DESCRIPTION
**Pull request by pjeanjean**
Should be merged only after eclipse/xtext#2530 is fixed.

### CODE REVIEW

File: org.eclipse.xtext.tests/src/org/eclipse/xtext/serializer/XtextSerializationTokensTest.java [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+ */
+public class XtextSerializationTokensTest extends AbstractXtextTests {
+
+	@Override
+	public void setUp() throws Exception {
+		super.setUp();
+		this.with(new XtextStandaloneSetup());
+	}
+
+	@Test
```
Comment (cdietrich) [inline org.eclipse.xtext.tests/src/org/eclipse/xtext/serializer/XtextSerializationTokensTest.java]: i assume the test should be set to ignore as long as there is no fix

**Comment by cdietrich** [comment]
@pjeanjean please signoff your commit.
also make sure you have the ECA https://www.eclipse.org/legal/ECA.php signed and the email fits the one in the signoff

**Comment by pjeanjean** [comment]
Done.

Do you want me to set the tests to Ignore? I wasn't considering this to be merged before having a fix so I didn't do it.

**Comment by cdietrich** [comment]
yes please do so

**Review by szarnekow — APPROVED** [review]

**Pull request merged**